unit_test/test_spdm_responder/capabilities.c

```diff
@@ -436,6 +436,7 @@ void test_spdm_responder_capabilities_case1(void **state)
     spdm_test_context = *state;
     spdm_context = spdm_test_context->spdm_context;
     spdm_test_context->case_id = 0x1;
+    spdm_context->connection_info.version = SPDM_MESSAGE_VERSION_10 << SPDM_VERSION_NUMBER_SHIFT_BIT;
```

Comment:
This is wrong.

Responder need use GET_CAPABILITIES to determine the version.

Reply:
Yes. I have changed the code.  The libspdm_rsp_capabilities.c use follow code to set version. So we can't use the spdm_context->connection_info.version before setting.
```
        if (local_ver == version) {
            spdm_context->connection_info.version = version << SPDM_VERSION_NUMBER_SHIFT_BIT;
            return TRUE;
        }
```